cmd/kubeapps-apis/plugins/resources/v1alpha1/server.go

```diff
@@ -58,10 +59,11 @@ type Server struct {
 	// We keep a restmapper to cache discovery of REST mappings from GVK->GVR.
 	restMapper meta.RESTMapper
 
-	// kindToResource is a function to convert a GVK to GVR. Can be replaced
-	// in tests with a dummy version using the unsafe helpers while the real
-	// implementation queries the k8s API for a REST mapper.
-	kindToResource func(meta.RESTMapper, schema.GroupVersionKind) (schema.GroupVersionResource, error)
+	// kindToResource is a function to convert a GVK to GVR with
+	// namespace/cluster scope information. Can be replaced in tests with a
+	// dummy version using the unsafe helpers while the real implementation
+	// queries the k8s API for a REST mapper.
+	kindToResource func(meta.RESTMapper, schema.GroupVersionKind) (schema.GroupVersionResource, meta.RESTScopeName, error)
```

Comment:
This update was required to be able to get/watch resources correctly. The current code only works for namespaced resources as, without this extra info, it assumes a namespace scope.